pkgs/development/python-modules/monty/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, nose, numpy, six, ruamel_yaml, msgpack-python, coverage, coveralls, pymongo, lsof }:
+
+buildPythonPackage rec {
+  pname = "monty";
+  version = "1.0.2";
+
+  # No tests in Pypi
+  src = fetchFromGitHub {
+    owner = "materialsvirtuallab";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0ss70fanavqdpj56yymj06lacgnknb4ap39m2q28v9lz32cs6xdg";
+  };
+
+  propagatedBuildInputs = [ nose numpy six ruamel_yaml msgpack-python coverage coveralls pymongo lsof ];
+  
+  preCheck = ''
+    substituteInPlace tests/test_os.py \
+      --replace 'def test_which(self):' '#' \
+      --replace 'py = which("python")' '#' \
+      --replace 'self.assertEqual(os.path.basename(py), "python")' '#' \
+      --replace 'self.assertEqual("/usr/bin/find", which("/usr/bin/find"))' '#' \
+      --replace 'self.assertIs(which("non_existent_exe"), None)' '#' \
+  '';
+
+  meta = with stdenv.lib; {
```

Comment:
Please provide a short description (in field `description`), too.

Reply:
@pSub  Done.